webcomponents/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="./dist/index.mjs" type="module"></script>
+    <link rel="stylesheet" href="./dist/style.css" />
```

Comment:
```suggestion
    <script src="./index.ts" type="module"></script>
    <link rel="stylesheet" href="./style/index.css" />
```

`<script src="">` should be `index.ts` for `npm run dev` to hot reload changes, same for the CSS file, although those styles does not affect ShadowDOM elements inside a WebComponent.